core/src/main/java/org/apache/iceberg/rest/requests/PlanTableScanRequest.java

```diff
@@ -107,6 +115,10 @@ public String toString() {
         .toString();
   }
 
+  public static Builder builder() {
```

Comment:
this makes sure that the builder is always created through a static method (and aligns it with the rest of the codebase)